v2/btree_test.go

```diff
@@ -0,0 +1,800 @@
+// Copyright 2014 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
```

Comment:
maybe this should be a public test, i.e. use `v2_test` package, e.g.: https://gist.github.com/SaveTheRbtz/3fa26a476b5cacd58d5662856860f8ae

(as a minor benefit this would also make Examples copy-pastable)